index.html

```diff
@@ -17,7 +20,7 @@ <h1>PlantList</h1>
 
   </nav>
 
-  <main>
+  <main class="wrapper">
```

Comment:
We might want to create a separate div for the wrapper inside the main here. I think that is the best practice.